build/sign.proj

```diff
@@ -23,7 +23,7 @@
       <FilesToSign Include="$(SolutionPackagesFolder)lucene.net\3.0.3\lib\NET40\Lucene.Net.dll">
         <Authenticode>3PartySHA2</Authenticode>
       </FilesToSign>
-      <FilesToSign Include="$(SolutionPackagesFolder)newtonsoft.json\9.0.1\lib\net45\Newtonsoft.Json.dll">
+      <FilesToSign Include="$(SolutionPackagesFolder)newtonsoft.json\13.0.1\lib\net45\Newtonsoft.Json.dll">
```

Comment:
nit: You can use the newtonsoft version variable.

Reply:
This proj file doesn't currently import `package.targets`, and while it wouldn't be difficult to do so, I don't feel like spinning another build.